.github/workflows/coverage-linux.yml

```
@@ -8,7 +8,6 @@ on:
       - 'benchmark/**'
       - 'deps/**'
       - 'doc/**'
-      - 'tools/**'
```

Have never used it myself, but based on https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-positive-and-negative-patterns-1, perhaps this would be better?

```suggestion
    paths:
      - '!**.md'
      - '!benchmark/**'
      - '!deps/**'
      - '!doc/**'
      - '!tools/**'
      - 'tools/inspector_protocol/**'
```

---

Whoops, that was supposed to be a suggestion to replace all of lines 6-11, not just line 11.